resources/codegen_inputs/svg.rs

```diff
@@ -0,0 +1,39 @@
+#![parse_module(read_fonts::tables::svg)]
+
+/// The [SVG](https://learn.microsoft.com/en-us/typography/opentype/spec/svg) table
+#[tag = "SVG "]
+table SVG {
```

Comment:
```suggestion
table Svg {
```

Please change this and rerun codegen and then we'll get this merged.

Reply:
Sorry! I read the comment wrong. 😅

Reply:
No worries :)